register.php

```diff
@@ -59,12 +59,14 @@ function () {
                 echo '<link rel="stylesheet" href="' . htmlspecialchars($timeTableStyle) . '"></link>';
                 $userId = htmlspecialchars(session('userdata.id'), ENT_QUOTES, 'UTF-8');
                 $ticketCacheExpiration = app()->make(Setting::class)->getSetting('itk-leantime-timetable.ticketCacheExpiration') ?: '1200';
+                $requireTimeRegistrationComment = app()->make(Setting::class)->getSetting('itk-leantime-timetable.requireTimeRegistrationComment') ?: '0';
```

Comment:
is 0 a sane default? what does 0 mean?

Reply:
should '0' be a int perhaps?

Reply:
Not really any reason to save it as int, so i changed it to just use the provided string "1" or setting it to "0"